pkg/scalers/pulsar_scaler.go

```diff
@@ -199,6 +200,18 @@ func parsePulsarMetadata(config *ScalerConfig) (pulsarMetadata, error) {
 	if err != nil {
 		return meta, fmt.Errorf("error parsing %s: %w", msgBacklogMetricName, err)
 	}
+
+	if auth != nil && auth.EnableOAuth {
+		if auth.OauthTokenURI == "" {
+			auth.OauthTokenURI = config.TriggerMetadata["oauthTokenURI"]
+		}
+		if auth.Scopes == nil && config.TriggerMetadata["scope"] != "" {
+			auth.Scopes = strings.Split(config.TriggerMetadata["scope"], " ")
```

Comment:
could you please trim the string first and change the format to accept *a comma separated list of scopes? To be consistent with the rest of the config.